apps/web/lib/getTeamMemberEmailFromCrm.ts

```diff
@@ -111,17 +113,20 @@ function getEnabledRoutingFormAppSlugFromQuery(query: ParsedUrlQuery) {
 /**
  * Uses the owner of the contact directly from CRM
  */
-async function getOwnerEmailFromCrm(eventData: EventData, email: string): Promise<string | null> {
-  const crmContactOwnerEmail = await getCRMContactOwnerForRRLeadSkip(email, eventData.metadata);
-  if (!crmContactOwnerEmail) return null;
+async function getOwnerEmailFromCrm(
+  eventData: EventData,
+  email: string
+): Promise<{ email: string | null; recordType: string | null; crmAppSlug: string | null }> {
```

Comment:
The change here is instead of returning just the attendee owner email, we also return the record type (contact, lead, account) and the app slug.